lib/money_oxr/rates_store.rb

```diff
@@ -35,7 +35,7 @@ def get_rate(iso_from, iso_to)
     end
 
     def loaded?
-      index.any?
+      rates.any?
```

Comment:
This is fine, but the new memory store has synchronization code we need to use too.